library/backend.cpp

```diff
@@ -436,9 +438,13 @@ static int dnet_cmd_backend_control_dangerous(struct dnet_net_state *st, struct
 		return -ENOENT;
 	}
 
-	if (cmd->size < sizeof(dnet_backend_control) + control->ids_count * sizeof(dnet_raw_id)) {
-		DNET_LOG_ERROR(node, "backend_control: command size is not enough for ids, state: {}",
-		               dnet_state_dump_addr(st));
+	const auto expected_size = sizeof(dnet_backend_control) +
+		control->ids_count * sizeof(dnet_raw_id) +
+		control->chunks_dir_len;
+
+	if (cmd->size < expected_size) {
```

Comment:
Why not `!=`?

Reply:
For future extensions.

Reply:
It seems strange after you have calculated definite expected size. Preliminary logic, and reliable calculated result is not used properly.

Reply:
Future extensions may only extend or keep the packet's size. This code should be ready for them.